cli/tests/e2e/snapshots/test_ci/test_full_run/autofix-azure-pipelines-overwrite-autodetected-variables/meta.json

```diff
@@ -11,6 +11,7 @@
     "commit_author_username": null,
     "commit_author_image_url": null,
     "commit_title": "Some other commit/ message",
+    "commit_timestamp": "1672552800",
```

Comment:
I always prefer human-readable timestamps, but that's personal preference. Either way, it would be helpful to include explicit timezone information here (if it's always UTC, call that out, if not, label with timezone)

Reply:
I've updated the code the match the same format that we use with findings, something similar to `2023-07-24T23:38:18`